.github/workflows/integrate.yaml

```diff
@@ -15,6 +15,7 @@ jobs:
           flutter-version: ${{ env.FLUTTER_VERSION }}
           cache: true
       - run: flutter pub get
+      - run: flutter gen-l10n
```

Comment:
@selfisekai can you just for fun change this to `dart run build_runner build --delete-conflicting-outputs`? Someone even recommended the funny workaround:

```sh
dart run build_runner build --delete-conflicting-outputs
flutter pub get
dart run build_runner build --delete-conflicting-outputs
```

Reply:
nevermind doesnt work